### PR TITLE
ScheduleState.hpp: forward Action::Actions

### PR DESCRIPTION
**Pull request by akva2**
Reduce hotness of Actions.hpp by 87% (145 files, 21 files instead of 166)

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this opm-simulators=4364 please

**Review by bska — APPROVED** [review]
Thanks&mdash;I'll merge into master.

File: opm/input/eclipse/Schedule/ScheduleState.hpp [hunk 1]
```diff
@@ -60,6 +59,9 @@ namespace {
 
 namespace Opm {
 
+    namespace Action {
+        class Actions;
+    };
```
Comment (bska) [inline opm/input/eclipse/Schedule/ScheduleState.hpp:64]: I didn't pay enough attention to this whilst reviewing, but the trailing semicolon on the closing brace of the namespace declaration is superfluous and will generate warnings in some elevated warning settings.

File: opm/input/eclipse/Schedule/Schedule.hpp [hunk 2]
```diff
@@ -45,6 +45,11 @@
 
 namespace Opm
 {
+    namespace Action {
+        class ActionX;
+        class PyAction;
+        class State;
+    };
```
Comment (bska) [inline opm/input/eclipse/Schedule/Schedule.hpp:52]: I didn't pay enough attention to this whilst reviewing, but the trailing semicolon on the closing brace of the namespace declaration is superfluous and will generate warnings in some elevated warning settings.

**Review by bska — COMMENTED** [review]
I just noticed a couple of minor issues when compiling this on my local machine.

**Pull request merged**